src/js/common/schemaform/review/ArrayField.jsx

```diff
@@ -164,7 +164,7 @@ class ArrayField extends React.Component {
                           ? <h5>New {uiSchema['ui:options'].itemName}</h5>
                           : null}
                       <SchemaForm
-                          data={item}
+                          pageData={item}
```

Comment:
These may end up needing `formData` as well, but I'd rather not squash all data to get `formData` here as well...